engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -160,23 +152,11 @@
 	HibernateValidatorConfiguration addValidatedValueHandler(ValidatedValueUnwrapper<?> handler);
 
 	/**
-	 * @return the default {@link org.hibernate.validator.spi.constraintdefinition.ConstraintDefinitionContributor}. Never {@code null}.
-	 *
-	 * @since 5.2
-	 */
-	ConstraintDefinitionContributor getDefaultConstraintDefinitionContributor();
-
-	/**
-	 * Registers the given {@code ConstraintDefinitionContributor} with the bootstrapped validator factory.
-	 *
-	 * @param contributor the {@code ConstraintDefinitionContributor} to register. Cannot be {@code null}.
-	 *
-	 * @return {@code this} following the chaining method pattern
+	 * @return the default {@link org.hibernate.validator.spi.cfg.ConstraintMappingContributor}. Never {@code null}.
 	 *
-	 * @hv.experimental This API is considered experimental and may change in future revisions
-	 * @since 5.2
+	 * @since 5.3
 	 */
-	HibernateValidatorConfiguration addConstraintDefinitionContributor(ConstraintDefinitionContributor contributor);
+	ConstraintMappingContributor getDefaultConstraintMappingContributor();
```

Comment:
I'd say it contributes defaults to constraint mappings :D 

More seriously, that's the equivalent of a method I removed, `getDefaultConstraintDefinitionContributor`. I had to, because `ConstraintDefinitionContributors` have been replaced with `ConstraintMappingContributors`. This method (and now its replacement) was used by `ValidatorFactoryImpl` to retrieve default constraint definitions. Currently, the default implementation is `ServiceLoaderBasedConstraintMappingContributor`, which performs automatic discovery of `ConstraintValidator`s. 

But, as you say below, this method is indeed useless in the public API. It could just exist in `ConfigurationImpl`. I will remove it from `HibernateValidatorConfiguration`.